beat/config.py

```diff
@@ -696,6 +700,35 @@ def init_waveforms(self, wavenames=['any_P']):
         for wavename in wavenames:
             self.waveforms.append(WaveformFitConfig(name=wavename))
 
+##Mahdi
+class PolarityConfig(Object):
+    stations_polarities = List.T(default=[])
+    name = String.T(default='pwfarrival', 
+                     optional=True,
+                     help="If not given, velocity model from Green's function will be extract")
+
+    channels = List.T(default=['Z'], 
+                     optional=True,
+                     help="If not given, velocity model from Green's function will be extract")
+
+    store_superdir = String.T(default='./', 
```

Comment:
here the gf_config needs definition instead of store_superdir

Reply:
please inherit from the NonlinearGFConfig create a PolarityGFConfig which is just like a SeismicGFConfig without:
```
    code = String.T(
        default='qssp',
        help='Modeling code to use. (qssp, qseis, comming soon: '
             'qseis2d)')
    sample_rate = Float.T(
        default=2.,
        help='Sample rate for the Greens Functions.')
    rm_gfs = Bool.T(
        default=True,
        help='Flag for removing modeling module GF files after'
```

Reply:
Done!